### PR TITLE
Document isMultiBlock param for block transforms

### PR DESCRIPTION
**Pull request by oandregal**
Documents the `isMultiBlock` param of block transformations.

### CODE REVIEW

File: docs/designers-developers/developers/block-api/block-transforms.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ A transformation of type `block` is an object that takes the following parameter
 - **blocks** _(array)_: a list of known block types. It also accepts the wildcard value (`"*"`), meaning that the transform is available to _all_ block types (eg: all blocks can transform into `core/group`).
 - **transform** _(function)_: a callback that receives the attributes and inner blocks of the block being processed. It should return a block object or an array of block objects.
 - **isMatch** _(function, optional)_: a callback that receives the block attributes and should return a boolean. Returning `false` from this function will prevent the transform from being available and displayed as an option to the user.
+- **isMultiblock** _(boolean, optional)_: whether the transformation can be applied when multiple blocks are selected. If this is true, the `transform` function will receive as paramaters an with the values of each selected block. False by default.
```
Comment (ntsekouras) [inline docs/designers-developers/developers/block-api/block-transforms.md:42]: You missed the word `array` in `will receive as paramaters an with...` and maybe replace `values` with `attributes`?

Also the API returns a second argument that is an array with the `InnerBlocks` of each selected block.

https://github.com/WordPress/gutenberg/blob/c1208188447b78e2b4e983c8b8f21a9f6c0d27f9/packages/blocks/src/api/factory.js#L463

**Review by ntsekouras — COMMENTED** [review]
Thanks @nosolosw - that's definitely helpful.

I recently worked on tranforms and missed the fact that documentation was lacking, by looking the source code.

File: docs/designers-developers/developers/block-api/block-transforms.md (hunk 1 above)
Comment (nosolosw) [reply docs/designers-developers/developers/block-api/block-transforms.md]: Good catch! Pushed a new wording. Is now clearer? Will welcome suggestions. Today, my brain has proven to have issues to enter _documentation mode_.

File: docs/designers-developers/developers/block-api/block-transforms.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ A transformation of type `block` is an object that takes the following parameter
 - **blocks** _(array)_: a list of known block types. It also accepts the wildcard value (`"*"`), meaning that the transform is available to _all_ block types (eg: all blocks can transform into `core/group`).
 - **transform** _(function)_: a callback that receives the attributes and inner blocks of the block being processed. It should return a block object or an array of block objects.
 - **isMatch** _(function, optional)_: a callback that receives the block attributes and should return a boolean. Returning `false` from this function will prevent the transform from being available and displayed as an option to the user.
+- **isMultiblock** _(boolean, optional)_: whether the transformation can be applied when multiple blocks are selected. If true, the `transform` function first parameter will be an array containing each selected block's attributes, and the second an array of each selected block's inner blocks. False by default.
```
Comment (ntsekouras) [inline docs/designers-developers/developers/block-api/block-transforms.md:42]:
```suggestion
- **isMultiblock** _(boolean, optional)_: whether the transformation can be applied when multiple blocks are selected. If true, the `transform` function's first parameter will be an array containing each selected block's attributes, and the second an array of each selected block's inner blocks. False by default.
```

**Review by ntsekouras — APPROVED** [review]
Thanks! 👍  - I think the `'s` is need and it's fine!

**Pull request merged**